apps/graphql/src/queries/__tests__/Itineraries.test.js

```diff
@@ -28,12 +28,12 @@ Object {
       "edges": Array [
         Object {
           "node": Object {
-            "id": "SXRpbmVyYXJ5OjE=",
+            "id": "SXRpbmVyYXJ5OmFzZGY=",
           },
         },
         Object {
           "node": Object {
-            "id": "SXRpbmVyYXJ5OjI=",
+            "id": "SXRpbmVyYXJ5OmFzZGY=",
```

Comment:
I not sure but, if it can make trouble in mocks but unique IDs are very important in GraphQL.